config/routes.rb

```diff
@@ -28,6 +28,8 @@ def matches?(request)
 
       get "/roles/:account/:kind/*identifier" => "roles#all_memberships", :constraints => QueryParameterActionRecognizer.new("all")
       get "/roles/:account/:kind/*identifier" => "roles#direct_memberships", :constraints => QueryParameterActionRecognizer.new("memberships")
+      put "/roles/:account/:kind/*identifier" => "roles#update_member", :constraints => QueryParameterActionRecognizer.new("members")
```

Comment:
This is following the pattern from V4:
https://github.com/conjurinc/authz/blob/9cefc77ebc98c7104b829e78a347b0708fa3664e/config/routes.rb#L22

I'm not opposed to changing it, but if we're making a change like that (that requires changing more in the UI), I'd probably push for changing the route path to something more like:
`/roles/:account/:kind/*identifier/members` for managing and searching the members of a role, rather than use the query parameter constraint.